docs/os/windows/windows-vm41.md

```diff
@@ -74,87 +57,78 @@ To install Qubes Windows Tools, follow instructions in [Qubes Windows Tools](htt
 
 - Create a VM named WindowsNew in [HVM](https://www.qubes-os.org/doc/hvm/) mode (Xen's current PVH limitations precludes from using PVH):
 
-  The Windows' installer requires a significant amount of memory or else the VM will crash with such errors:
-
-  `/var/log/xen/console/hypervisor.log`:
-
-  ~~~
-  p2m_pod_demand_populate: Dom120 out of PoD memory! (tot=102411 ents=921600 dom120)
-  (XEN) domain_crash called from p2m-pod.c:1218
-  (XEN) Domain 120 (vcpu#0) crashed on cpu#3:
-  ~~~
-
-  So, increase the VM's memory to 4096MB (memory = maxmem because we don't use memory balancing).
-
-  ~~~
-  qvm-prefs WindowsNew memory 4096
-  qvm-prefs WindowsNew maxmem 4096
-  ~~~
-
-  Disable direct boot so that the VM will go through the standard cdrom/HDD boot sequence:
-
-  ~~~
-  qvm-prefs WindowsNew kernel ''
-  ~~~
-
-  A typical Windows installation requires between 25GB up to 60GB of disk space depending on the version (Home/Professional/...). Windows updates also end up using significant space. So, extend the root volume from the default 10GB to at least 60GB (note: it is straightforward to increase the root volume size after Windows is installed: simply extend the volume again in dom0 and then extend the system partition with Windows's disk manager).
+  In order to create the new qube, select the command Qube -> New Qube in the Qube Manager::
+  - Name: `WindowsNew`, Color: `orange` (for a standalone qubes, `black` for a template)
+  - Type: `StandaloneVM (fully persistent)` or `TemplateVM (template home, persistent root)`
+  - Template: `(none)`
```

Comment:
This could be misleading ; maybe use
`- Template: '' (no template defined)`

Reply:
The Qube Manager does not support empty values for kernel and template but uses the option (none) instead.

Reply:
Ah OK - I had missed the context was within Qube Manager. Thanks !